Earthfile

```diff
@@ -54,7 +54,7 @@ lint:
             echo "$output" ; \
             exit 1 ; \
         fi
-    RUN output="$(goimports -d . 2>&1)" ; \
+    RUN output="$(goimports -d . 2>&1)" | grep -v '.*\.pb\.go' ; \
```

Comment:
Should the grep be within the `$(...)`?

Reply:
I'm not sure how this was working, it doesn't work inside the braces; i'll change it to use find instead.